WSL/compare-versions.md

```diff
@@ -44,7 +44,18 @@ For example, when storing your WSL project files:
 
 Project files that you are working with using a WSL distribution (like Ubuntu) must be in the Linux root file system to take advantage of faster file system access.
 
-You can access your Linux root file system with Windows apps and tools like File Explorer. Try opening a Linux distribution (like Ubuntu), be sure that you are in the Linux home directory by entering this command: `cd ~`. Then open your Linux file system in File Explorer by entering *(don't forget the period at the end)*: `explorer.exe .`
+### Accsess the Linux file system and run Windows commands inside distribution
```

Comment:
Blocking this based on the suggested changes and the fact that this content is in the `compare-versions.md` doc but doesn't compare the WSL versions.